TYPO3.Neos/Resources/Private/TypoScript/Prototypes/Menu.ts2

```diff
@@ -6,7 +6,7 @@ prototype(TYPO3.Neos:Menu) < prototype(TYPO3.TypoScript:Template) {
 	node = ${node}
 	items = ${this.items}
 
-	entryLevel = 1
+	entryLevel = ${this.startingPoint ? 0 : 1}
```

Comment:
Not totally sure about this. Would also work of course. But feels for me like too much magic. The way it is right now the entryLevel will just be set with a better suiting default value. If you go for a process and want to override the entryLevel if an startingPoint is present you also have to override the process. For me a process is used if i want to process a given value in some way. But in this case it's just try to find a good default value but don't touch the value afterwards it is set.

But I would also be fine with a process. So ...?